components/layout/Search.vue

```diff
@@ -43,20 +97,38 @@
                             <div class="rounded-3 w-100" v-if="selectedItem">
                                 <div>
                                     <h4>
-                                        <CommonTaskIcon v-if="selectedItem.cls" :cls="selectedItem.cls" />
-                                        <component class="icon-wrapper" v-else :is="iconByType(selectedItem.type)" />
+                                        <CommonTaskIcon
+                                            v-if="selectedItem.cls"
+                                            :cls="selectedItem.cls"
+                                        />
+                                        <component
+                                            class="icon-wrapper"
+                                            v-else
+                                            :is="iconByType(selectedItem.type)"
+                                        />
                                         {{ selectedItem.title }}
                                     </h4>
                                     <div class="slug">
                                         <span
-                                            :class="{first: index === 0}"
-                                            v-for="(item, index) in breadcrumb(selectedItem.url)" :key="item">
-                                                {{ item }}
+                                            :class="{ first: index === 0 }"
+                                            v-for="(item, index) in breadcrumb(
+                                                selectedItem.url,
+                                            )"
+                                            :key="item"
+                                        >
+                                            {{ item }}
                                         </span>
                                     </div>
-                                    <span class="badge bg-info">{{selectedItem.type.charAt(0).toUpperCase() + selectedItem.type.slice(1).toLowerCase()}}</span>
+                                    <span class="badge bg-info">{{
+                                        selectedItem.type
+                                            .charAt(0)
+                                            .toUpperCase() +
+                                        selectedItem.type.slice(1).toLowerCase()
+                                    }}</span>
```

Comment:
I find the old inline style easier to read (simple string transform + concat like that can be one-lined imo)